src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER = 100;
```

Comment:
Variable's name should explain it's purpose
```suggestion
    private static final int NUMBER_LIMIT = 100;
```